src/io/openshift/plugins/analytics.groovy

```diff
@@ -1,10 +1,29 @@
 package io.openshift.plugins
 
 import io.openshift.Events
+import io.openshift.Utils
 
 def register() {
     Events.on("build.pass") {
         e, a -> echo "invoking bayesian analytics $e $a"
+        def image = config.runtime() ?: ''
+        if(image != "") {
+            def url = Utils.shWithOutput(this, "git config remote.origin.url")
+            echo "URL:$url"
+            echo "Image:$image"
+            try {
+                retry(3) {
+                    def response = bayesianAnalysis url: 'https://bayesian-link', gitUrl: url, ecosystem: image
```

Comment:
I actually wonder if we can just use `scm` here. have you tried print what scm  returns? 

see: https://stackoverflow.com/questions/38254968/how-do-i-get-the-scm-url-inside-a-jenkins-pipeline-or-multibranch-pipeline/38255364#38255364